app.rb

```diff
@@ -1,9 +1,31 @@
 require 'sinatra/base'
+require './lib/peeps'
+require './lib/setup'
 
 class Chitter < Sinatra::Base
+  enable :sessions, :method_override
+  
   get '/test' do
     'Test page'
   end
 
+  get '/peeps/new' do
+    erb(:peeps_new)
+  end
+  post '/peeps' do
+    Peep.add(params[:peep])
+    redirect '/peeps'
+  end
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb(:index)
+  end
+
+  get '/peeps/:id' do
+    @peeps = Peep.filter(params[:filter])
+    erb(:show_filtered)
+  end
+
```

Comment:
Maybe have more descriptive names? Just so it's easier for an outsider to follow, but that's being super nit-picky.